modules/news/news.service.js

```diff
@@ -5,8 +5,21 @@ const {
 } = require('../../error-messages/news.messages');
 
 class NewsService {
-  async getAllNews() {
-    return await News.find();
+  async getAllNews({ skip, limit }) {
+    const items = await News.find()
+      .skip(skip)
+      .limit(limit);
+
+    if (!items) {
+      throw new Error(NEWS_NOT_FOUND);
```

Comment:
This check doesn't have any sense because `items` are always going to be an array. Please, get rid of it, even empty array is a valid response

Reply:
You're right! Removed!